modules/ROOT/pages/spring-data-neo4j.adoc

```diff
@@ -1,318 +1,478 @@
+The Spring Initializr will take care of creating the project structure for you, with the basic files and settings in place for the selected build tool.
+
+=== Other dependencies
+
+If you are looking at the project in Github, you might notice that there are some other dependencies in the `pom.xml`.
+A couple are for adding tests to the project, then one dependency for developer tools, and a couple more for test containers.
+
+More information on the testing functionality can be found in the https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/#sdn.testing[documentation^].
+
+.Testing and dev tools dependencies
 [source,xml,subs="verbatim,attributes"]
 ----
-<dependencies>
-    <dependency>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-data-neo4j</artifactId>
-	</dependency>
-</dependencies>
+<dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-test</artifactId>
+    <scope>test</scope>
+</dependency>
+<dependency>
+    <groupId>io.projectreactor</groupId>
+    <artifactId>reactor-test</artifactId>
+    <scope>test</scope>
+</dependency>
+<dependency>
+    <groupId>org.testcontainers</groupId>
+    <artifactId>junit-jupiter</artifactId>
+    <version>{testcontainers-version}</version>
+    <scope>test</scope>
+</dependency>
+<dependency>
+    <groupId>org.testcontainers</groupId>
+    <artifactId>neo4j</artifactId>
+    <version>{testcontainers-version}</version>
+    <scope>test</scope>
+</dependency>
+<dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-devtools</artifactId>
+    <scope>runtime</scope>
+    <optional>true</optional>
+</dependency>
+<dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-devtools</artifactId>
+    <scope>runtime</scope>
+    <optional>true</optional>
+</dependency>
+----
+
+[#adding-config]
+== Adding configurations
+
+Now, we need to add a few configurations to connect to the database.
+We can find the `application.properties` file and configure what we need.
+
+[source,properties]
+----
+spring.neo4j.uri=neo4j://localhost:7687
+spring.neo4j.authentication.username=neo4j
+spring.neo4j.authentication.password=secret
 ----
 
-[#node-entities]
-== Entities are Nodes
+[NOTE]
+--
+You will need to adjust the password to whatever you set when you created your instance of Neo4j.
+--
 
-The entities outline our main objects in the application.
-In our graph, they will be the nodes.
+The first three lines are our Neo4j database URI and credentials.
+The username and password you enter here should match for your individual database.
+This is the bare minimum of what you need to connect to a Neo4j instance.
 
-image::{img}GraphModel.png[]
+We do not need to add any configuration for the driver with the SDN starter, as the repositories will be automatically enabled with it.
```

Comment:
Please rephrase this. It is exactly the opposite: Those properties above configure the Neo4j driver (bean) SDN depends on not Spring Data Neo4j.

Reply:
s/any configuration/any other configuration/ ?

Reply:
Ok with _any other configuration_ but somehow on the other side not really pleased that the configuration refers to the SDN starter. It should be the driver auto configuration.

Reply:
done in fc88b3aa4bbae2a0dc163d11141ccee2a1dfc259

Reply:
_driver auto configuration_ not starter. The linked starter is ours (for Spring Boot 2.2/2.3) but the auto configuration is the magic ingredient for Spring Boot 2.4 (I think those we are targeting here) in Spring Boot itself.
Please no link and don't call it starter ;)

Reply:
oopsie, my bad

Reply:
fixed in 92df2ba154c2fbef831d3b90c7cdfd8c7c531e19